e2e/core/basesuite.go

```diff
@@ -87,6 +87,18 @@ func (s *BaseSuite) SendTxBlock(msg sdk.Msg, from keys.KeyManager) *sdk.TxRespon
 	return getTxRes.TxResponse
 }
 
+func (s *BaseSuite) SimulateTx(msg sdk.Msg, from keys.KeyManager) (txRes *tx.SimulateResponse) {
+	mode := tx.BroadcastMode_BROADCAST_MODE_BLOCK
```

Comment:
it seems we don't support `block` mode now

Reply:
replaced with `sync` mode